docs/user-guide/mvd-configuration.md

```diff
@@ -6,54 +6,70 @@ When you install Zowe&trade;, the App Framework is configured as a Mediation Lay
 
+### zss configuration
 
+ZSS shares some parameters in common with the app-server, so you can consult the above json-schema document to find out which parameters are valid within `components.zss` of the Zowe configuration file. However, some parameters within the app-server schema are not used by ZSS, such as the `node` section. A ZSS-centric schema will be available soon.
+
+## Environment variables
+
+In the latest version of Zowe, `instance.env` is no longer used. However, some environment variables that could be specified within v1 can still be set within v2 in the `zowe.environments` section of the server configuration file. Environment variables starting with `ZWED_` map to values that can be specified within `components.app-server` and `components.zss` so they are redundant, but you can refer to the above json-schema document to see which values are useful or deprecated.
+
+
+## Configuring the framework as a Mediation Layer client
 
-The port number for the API Mediation Layer is the value of the *components.gateway.port* variable in the same file.
+The App Server and ZSS automatically register to the API Mediation Layer when present.
+If this is not desired, it can be disabled by setting the properties `components.app-server.mediationLayer.server.enabled=false` for app-server and `components.zss.mediationLayer.enabled=false` for ZSS.
 
 ## Setting up terminal app plugins
 
 Follow these optional steps to configure the default connection to open for the terminal app plugins.
 
 ### Setting up the TN3270 mainframe terminal app plugin
 
-`_defaultTN3270.json` is a file in `tn3270-ng2/`, which is deployed during setup. Within this file, you can specify the following parameters to configure the terminal connection:
+The file `_defaultTN3270.json` within the `tn3270-ng2` app folder `/config/storageDefaults/sessions/` is deployed to the [configuration dataservice](../extend/extend-desktop/mvd-configdataservice.md) when the app-server runs for the first time. This file is used to tell the terminal what host to connect to by default. If you'd like to customize this default, you can edit the file directly within the configuration dataservice `<components.app-server.instanceDir>/org.zowe.terminal.tn3270/sessions/_defaultTN3270.json`. Or you can open the app, customize a session within the UI, click the save icon (floppy icon) and then copy that file from `<components.app-server.usersDir>/<your user>/org.zowe.terminal.tn3270/sessions/_defaultTN3270.json` to `<components.app-server.instanceDir>/org.zowe.terminal.tn3270/sessions/_defaultTN3270.json`. Either way, you will see a file with the following properties:
```

Comment:
components.app-server doesn't have an instanceDir variable by default in sample zowe.yaml, so we should say what the default location is: looks to be 
`<workspaceDir>/app-server/ZLUX/pluginStorage/org.zowe.terminal.tn3270/sessions/_defaultTN3270.json`

?

Reply:
it will. work Joe and I are doing.